### PR TITLE
changed opacity

### PR DESCRIPTION
**Pull request by fbawaney**

### CODE REVIEW

File: styles/main.less [hunk 1]
```diff
@@ -9,7 +9,7 @@ html,body {
 }
 
 body {
-  background: url('../images/habana_vieja_blue.jpg') 100%;
```
Comment (ocalvet) [inline styles/main.less:12]: Why did you delete this?

Comment (fbawaney) [inline styles/main.less:12]: too blue
